### ISSUE TITLE
REST Api returns "405 Method not allowed"

### ISSUE THREAD

**Issue by ratcashdev** [apache/pulsar#2193]
#### Expected behavior
Running apachepulsar/pulsar:2.0.1-incubating standalone docker image.

GET http://127.0.0.1:8080/admin/persistent/default/public/test-topic1/stats
HTTP 200

#### Actual behavior

GET http://127.0.0.1:8080/admin/persistent/default/public/test-topic1/stats
HTTP 405 METHOD NOT ALLOWED

#### Steps to reproduce

```
docker run -it \
  -p 6650:6650 \
  -p 8080:8080 \
  -v $PWD/data:/pulsar/data \
  apachepulsar/pulsar:2.0.1-incubating \
  bin/pulsar standalone
```
, afterwards in a browser window enter 
`http://127.0.0.1:8080/admin/persistent/default/public/test-topic1/stats`

#### System configuration
**Pulsar version**: 2.0.1-incubating running in docker on a windows host

**Comment by merlimat** [comment]
Thanks for reporting @ratcashdev. I'll get a quick fix for the docs and the dashboard code.

**Comment by merlimat** [comment]
@ratcashdev Created https://github.com/apache/incubator-pulsar/pull/2196 for the doc fixes. Check the warning around the advertised address in the PR.

Please also be aware of #2088. This is scheduled to be fixed in 2.1 release. For now, I'd recommend to just build the docker image locally.

**Comment by sijie** [comment]
This bug has been fixed by #2088 and documentation is improved by #2174 and #2196
